src/hockeypuck/server/settings.go

```diff
@@ -230,6 +232,7 @@ func DefaultSettings() Settings {
 		Version:        Version,
 		BuiltAt:        BuiltAt,
 		MaxResponseLen: DefaultMaxResponseLen,
+		EnableVHosts:   DefaultEnableVHosts,
```

Comment:
You don't need to explicitly set a toml boolean default, because the toml reader sets all named booleans to false by default.

Reply:
This is now fixed :)